docs/Project.toml

```diff
@@ -0,0 +1,5 @@
+[deps]
+DiffEqJump = "c894b116-72e5-5b58-be3c-e6d8d4ac2b12"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
For now we don’t need plots or revise as delendencies.

Reply:
@isaacsas Thanks. Yeah, I can remove the `settings.json` file. I think that must have snuck in there before I updated the `.gitignore` file. In terms of the `Plots` package, we probably both agree that docstrings don't need plots :). But for the `LabelledArrays.jl` package I also had to incorporate the tutorials into the documentation package--so that the tutorials will go in the combined docs. In that case, I imagine I would need the Plots. I can take a look at other examples in the new combined docs and see if other folks are also including the tutorials--just to confirm.

Reply:
I’m planning to handle the restructuring of the DiffEqJump tutorial so don’t worry about that. For now the plots in the tutorial are all pre-generated so don’t need plots.jl to get made.

Reply:
Sounds good. I will take that out.

Reply:
I removed `Plots`, `Revise`, and `vscode` folder.